tests/test.sh

```diff
@@ -8,8 +8,20 @@ set -e
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"  # https://stackoverflow.com/a/246128
 
 if [ -z ${NO_APT_INSTALL+x} ]; then
- sudo DEBIAN_FRONTEND=noninteractive apt-get -qq update
- sudo DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends openjdk-8-jdk python3 python3-pip docker.io php ruby2.7 dotnet-sdk-6.0
+   if [ "$(uname -m)" = "aarch64" ]; then
+    curl -SL -o dotnet.tar.gz https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/dotnet-sdk-latest-linux-arm64.tar.gz
```

Comment:
For all non-apt installers, please make sure they handle re-runs of the script. For example this installation of `dotnet` fails because `ln -s` fails as `dotnet` already exists. The installer logic can check if the installed is already here, and skip, like apt does when you run `apt install -y`.